src/main/java/teammates/ui/controller/InstructorEditInstructorFeedbackPageAction.java

```diff
@@ -34,9 +34,9 @@ protected ActionResult execute() throws EntityDoesNotExistException {
 
         String moderatedQuestionId = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_MODERATED_QUESTION_ID);
 
-        Assumption.assertNotNull(Const.StatusMessages.NULL_POST_PARAMETER_MESSAGE, courseId);
-        Assumption.assertNotNull(Const.StatusMessages.NULL_POST_PARAMETER_MESSAGE, feedbackSessionName);
-        Assumption.assertNotNull(Const.StatusMessages.NULL_POST_PARAMETER_MESSAGE, instructorUnderModerationEmail);
+        Assumption.assertPostParamNotNull(Const.StatusMessages.NULL_POST_PARAMETER_MESSAGE, courseId);
+        Assumption.assertPostParamNotNull(Const.StatusMessages.NULL_POST_PARAMETER_MESSAGE, feedbackSessionName);
+        Assumption.assertPostParamNotNull(Const.StatusMessages.NULL_POST_PARAMETER_MESSAGE, instructorUnderModerationEmail);
```

Comment:
It may be better to standardise using parameter name as the first argument to this method, instead of using the status message.